sdk/signalr/azure-messaging-webpubsubservice/examples/send_messages.py

```diff
@@ -62,6 +62,14 @@
     print('The group exists!')
 
 
+request = build_grant_permission_request('skalman', 'sendToGroup', '7')
+response = client.send_request(request)
+response.read()
+print(response.content)
```

Comment:
Does the `read()` method not also return the content?

Reply:
After reading the rest of the code I see that it does - in which case I think it would look more natural in the samples to simply print the returned value?

Reply:
response.read reads the response into memory. but since this isn't a streamed response, it's not needed in the samples any more